cloudflare.php

```diff
@@ -42,6 +42,30 @@
     }
 }
 
+// Call when the Plugin is activated in server. 
+function cloudflare_activate()
+{
+    global $wp_version;
+    $min_wp_version = '3.1';
+    $min_php_version = '5.3';
```

Comment:
move these into `const`s at the top of the file and add a trialing 0 to the end just to be explicit.